airflow/airflow_test.go

```diff
@@ -77,6 +77,33 @@ func (s *Suite) TestInit() {
 	}
 }
 
+func (s *Suite) TestTemplateInit() {
```

Comment:
could we cover negative test cases as well

Reply:
I've included the negative test case now.